src/Proto3/Wire/Encode.hs

```diff
@@ -77,8 +78,7 @@ module Proto3.Wire.Encode
       -- * Embedded Messages
     , embedded
       -- * Packed repeated fields
-    , packedVarints32
```

Comment:
Would it be possible to keep the original `packedVarints{32,64}` utilities so that this is not a breaking API change?

Reply:
The 32/64 distinction turned out to be unnecessary, but making it specific to vectors is potentially significant.  I could leave out some of those more aggressive optimizations on a first pass, or provide an alternative function.

Reply:
In 5fefcb2713f0e08d0d170cfa09397b314bce16f9 I restored the old `packedVarints` function for backward compatibility, but added vector-specific variants that should be faster.  Does that resolve this issue?